packages/eas-cli/src/commands/submit.ts

```diff
@@ -33,6 +33,7 @@ interface RawCommandFlags {
   verbose: boolean;
   wait: boolean;
   'non-interactive': boolean;
+  'verbose-fastlane'?: boolean;
```

Comment:
```suggestion
  'verbose-fastlane': boolean;
```

If it defaults to `false` I don't think it's optional